pkg/edition/java/lite/forward.go

```diff
@@ -58,17 +61,25 @@ func Forward(
 		return
 	}
 
-	log.Info("forwarding connection", "backendAddr", netutil.Host(dst.RemoteAddr()))
+	// Extract the backendAddr with the port and the IP separately
+	backendAddrWithPort := dst.RemoteAddr().String()
+	backendIP, _, _ := net.SplitHostPort(backendAddrWithPort)
+
+	// Include the strategy name in the log
+	strategyName := route.Strategy
+
+	log.Info("forwarding connection", "clientAddr", netutil.Host(src.RemoteAddr()), "virtualHost", ClearVirtualHost(handshake.ServerAddress), "protocol", proto.Protocol(handshake.ProtocolVersion).String(), "route", route.Host, "backendAddr", backendAddrWithPort, "backendAddr", backendIP, "strategy", strategyName)
+
 	pipe(log, src, dst)
 }
 
 // errAllBackendsFailed is returned when all backends failed to dial.
 var errAllBackendsFailed = errors.New("all backends failed")
 
 // tryBackends tries backends until one succeeds or all fail.
-func tryBackends[T any](next nextBackendFunc, try func(log logr.Logger, backendAddr string) (logr.Logger, T, error)) (logr.Logger, T, error) {
+func tryBackends[T any](log logr.Logger, next nextBackendFunc, try func(log logr.Logger, backendAddr string) (logr.Logger, T, error)) (logr.Logger, T, error) {
 	for {
-		backendAddr, log, ok := next()
```

Comment:
do not remove log returned from next(). We need to return this log as its contains more information that the log passed to tryBackends